resources/setups/common/base_config.json

```diff
@@ -49,7 +49,18 @@
       "enabled": true
     },
     "dot1x": {
-      "enabled": true
+      "enabled": true,
+      "interface": "eth0",
+      "radius": {
```

Comment:
is this for the (potentially) remote radius server?  If so, maybe key with "radius_server"... ?

Reply:
Changed.